### PR TITLE
update_readme

### PR DESCRIPTION
**Pull request by DmitriiTsy**

### CODE REVIEW

**Review by james-le-twelve-labs — APPROVED** [review]
Looks good!

File: README.md
Comment (TravisCouture) [inline README.md:1]: We shouldn't include the raw API key in the README.

Comment (james-le-twelve-labs) [reply README.md:1]: CC @DmitriiTsy

Comment (DmitriiTsy) [reply README.md:1]: @james-le-twelve-labs @TravisCouture 

Do we want to use a secrets management service to share keys?

Comment (james-le-twelve-labs) [reply README.md:1]: Yes, I think that's a good idea.

**Review by james-le-twelve-labs — APPROVED** [review]
Looking good now

**Pull request closed without merging**